src/services/rest/http-client.js

```diff
@@ -3,6 +3,8 @@ import url from 'url';
 // see getDefaultHTTPModule for additional imports due to webpack
 
 const IS_NODE = typeof window === 'undefined';
+const HTTP_200_OK_RANGE_START = 200;
```

Comment:
Maybe a list of explicitly "accepted" status codes (e.g. `ACCEPTED_HTTP_STATUSES`)? They're not that many: https://httpstatuses.com/ - As an example, is `204` really acceptable here (don't we expect the response to contain something?)